docs/API_POLICY.md

```diff
@@ -0,0 +1,30 @@
+# Hub API Policy
+This document proposes a policy regarding making API updates in this repo.
+
+There are 2 types of APIs:
+1. Versioned (For External Users): These are the APIs which are for users to use and will have support provided by the Hub team. These are the APIs which return tekton resources data. These are versioned APIs, the version of the APIs will be prefixed to the API route. 
+Eg. resource API - /v1/resources
+2. Internal APIs: These are the APIs which are specifically used for Hub UI and hub services. These are not exposed for integration.
+Eg. Login, rating, Catalog refresh APIs, etc.
+
+### User facing APIs
+- These APIs can be used for integration for users.
+- These APIs are versioned and have prefixed their version. Eg. /v1
+
+#### Additive changes
+Additive changes are changes that add to the API and do not cause problems for users of previous versions of the API.
+These changes must be approved by at least 2 OWNERS.
+
+#### Backwards incompatible changes
+Backwards incompatible changes change the API, eg. removing a field in API response or removing an API.  These changes will mean that folks using a previous version of the API will need to adjust their usage in order to use the new version or an alternate API.
```

Comment:
Don't you think this should first follow the process of deprecation first and then removal, or is it that there can be some scenario where we have direct removal